### PR TITLE
Remove unused packages and tito whitelists

### PR DESCRIPTION
**Pull request by ekohl**
Please carefully look if this is correct. jenkins_api client was used by foreman_pipeline but that was never imported to this repository and not rebuilt for Ruby 2.4 without breaking repoclosure.

### CODE REVIEW

**Comment by ekohl** [comment]
obal doesn't really understand package removal :)

**Comment by ehelms** [comment]
I think we should also drop -- tfm-rubygem-foreman_pipeline since these were dependencies of it (and I believe it is unmaintained)

**Comment by ekohl** [comment]
Ah, it is in comps but the spec file isn't there.

**Comment by ekohl** [comment]
Updated.

File: rel-eng/tito.props [hunk 1]
```diff
@@ -395,8 +394,7 @@ whitelist = katello-repos katello-host-tools
 [katello-nightly-rhel7]
 disttag = .el7
 scl = tfm
-whitelist = rubygem-robotex
```
Comment (ehelms) [inline rel-eng/tito.props]: This package is still used

Comment (ehelms) [reply rel-eng/tito.props]: Oh I see it down below, carry on

**Pull request merged**